src/test/java/teammates/test/pageobjects/FeedbackSubmitPage.java

```diff
@@ -125,4 +125,8 @@ public void linkOnCommentsLink() {
         AppPage.getNewPageInstance(browser, StudentCommentsPage.class);
         
     }
+    
+    public void waitForCellHoverToDisappear() {
+        waitForElementToDisappear(By.className("cell-hover"));
```

Comment:
I did manually test to see if it works by stepping through the code though. I added a break point at line 150 and added the `cell-hover` class to the element before resuming the test. The test paused, so I hovered over and off the cell to remove `cell-hover`, and the test then proceeded.